packages/superset-ui-plugin-chart-icicle-event/README.md

```diff
@@ -0,0 +1,39 @@
+## @superset-ui/plugin-chart-icicle
+
+## WIP
+
+This is a work in progress with the design being finalized.
+
+This plugin provides Icicle Event Chart for Superset.
+
+### Usage
+
+Configure `key`, which can be any `string`, and register the plugin. This `key` will be used to lookup this chart throughout the app.
+
+```js
+import IcicleEventVizPlugin from '@superset-ui/plugin-chart-icicle-event';
+
+new IcicleEventVizPlugin()
+  .configure({ key: 'icicle' })
+  .register();
+```
+
+Then use it via `SuperChart`. See [storybook](https://apache-superset.github.io/superset-ui-plugins/?selectedKind=plugin-chart-word-cloud) for more details.
```

Comment:
Might want to update the link in a later PR since IcicleEventChart is not available yet.